problems/strings.js

```diff
@@ -87,31 +131,62 @@ function removeLastWord() {}
  * @returns {string}
  */
 
-function everyOtherLetter() {}
+function everyOtherLetter(str) {
+  let even = ""
+  for(let i =0; i< str.length; i++) {
+    if( i%2 === 0) {
+      even += str[i];
+    }
+  }
+  return even
+
+
+}
 
 /**
  * Takes in a string and returns a new string that has all the vowels removed.
  * @param {string}
  * @returns {string}
  */
-function disemvowel() {}
-
+function disemvowel(str) {
+  let vowArr = ["a","e", "i", "o", "u", "A", "E", "I","O","U"]
+  let newStr = "";
+  for(let i = 0; i < str.length; i += 1) {
+    if(!vowArr.includes(str[i])) {
```

Comment:
.includes() can also be used on a string. So, this will still work if you write let vowStr = "aeiouAEIOU" on line 152.